tests/test_diffusers.py

```diff
@@ -1841,3 +1841,62 @@ def test_train_text_to_image_sdxl_lora(self):
 
             # save_pretrained smoke test
             self.assertTrue(os.path.isfile(os.path.join(tmpdir, "pytorch_lora_weights.safetensors")))
+
+
+class TrainControlNet(TestCase):
```

Comment:
Tests added take 1 min and 30s to run, should I add it under slow or fast test for diffusers in Makefile?

Reply:
@regisss  could you comment on this?

Reply:
Please add them as slow tests, that's what we do for testing examples. You may also need to add the `slow` decorator for these tests (see [here](https://github.com/huggingface/optimum-habana/blob/4a38cfe5fd02ac75a7298628a7173c82a8760386/tests/test_diffusers.py#L543) for an example).
1min30s is on Gaudi2 I guess?

Reply:
Will do. Yes the slow decorator will be needed.
Yes its on Gaudi2.

Reply:
Updated